src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -34,7 +35,11 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+            backgroundColor: isSelect
+              ? isAnon
+                ? COLORS.anon_primary + backgroundOpacity15PercentHexCode
+                : COLORS.signed_primary + backgroundOpacity15PercentHexCode
+              : COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for setting the background color is a bit complex and hard to read. Consider simplifying it by creating a helper function that returns the appropriate color based on `isSelect` and `isAnon`. This will improve readability and maintainability.
```diff
- backgroundColor: isSelect
-   ? isAnon
-     ? COLORS.anon_primary + backgroundOpacity15PercentHexCode
-     : COLORS.signed_primary + backgroundOpacity15PercentHexCode
-   : COLORS.white

+ backgroundColor: getBackgroundColor(isSelect, isAnon)
```
And then define the `getBackgroundColor` function like this:
```javascript
const getBackgroundColor = (isSelect, isAnon) => {
  if (!isSelect) return COLORS.white;
  const colorBase = isAnon ? COLORS.anon_primary : COLORS.signed_primary;
  return colorBase + backgroundOpacity15PercentHexCode;
};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->